addons/point_of_sale/models/account_move.py

```diff
@@ -10,6 +10,17 @@ class AccountMove(models.Model):
     pos_order_ids = fields.One2many('pos.order', 'account_move')
     pos_payment_ids = fields.One2many('pos.payment', 'account_move_id')
 
+    def _compute_payments_widget_reconciled_info(self):
+        payments_widget_vals = super()._compute_payments_widget_reconciled_info()
+        for move in self:
+            if move.invoice_payments_widget:
+                for reconciled_vals in move.invoice_payments_widget['content']:
+                    move_id = self.env['account.move'].browse(reconciled_vals['move_id'])
+                    if move_id.sudo().pos_payment_ids:
+                        pos_payment = move_id.sudo().pos_payment_ids
+                        reconciled_vals['pos_payment_name'] = pos_payment.payment_method_id.name
+        return payments_widget_vals
```

Comment:
I don't understand why we would show the payment method when linked to POS but not on the standard accounting module. To me, it should be either a fix/imp of the accounting report or not implemented at all. Did you discuss about that with product owners?

Reply:
hi @smetl ,
i haven't discussed with product owners, but it has been initially added by the pos app by this commit: https://github.com/odoo/odoo/commit/6420b6d778ad24b9b441a7158ba85bc96ba5b60f by @caburj